doc/symfony-cache-configuration.rst

```diff
@@ -182,6 +182,8 @@ options through the constructor:
 
 * **anonymous_hash**: Hash used for anonymous user. This is a performance
   optimization to not do a backend request for users that are not logged in.
+  By default, the header is skipped. If you specify a header, that header is
+  used.
```

Comment:
This is a bit unclear. What’s the header you’re talking about being skipped or used?